test/test_transforms.jl

```diff
@@ -5,7 +5,7 @@ using Base.Test
 
 using Celeste: Types, SampleData, Transform, SensitiveFloats
 import Celeste: ModelInit, ElboDeriv
-
+using Compat
```

Comment:
Removing them before seemed like a good thing to do...I didn't realize how soon we'd need them again